src/main.rs

```diff
@@ -14,6 +20,21 @@ impl fmt::Display for MissingScriptError {
 
 impl Error for MissingScriptError {}
 
+#[derive(Debug)]
+struct FailureStatus(ExitStatus);
+
+impl fmt::Display for FailureStatus {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if let Some(code) = self.0.code() {
+            write!(f, "Process exits with non-zero status code {:?}", code)
+        } else {
+            write!(f, "An unknown problem occurred during execution of process")
+        }
+    }
+}
```

Comment:
I guess it makes sense to print additional error when `pn run` exits with non-0 (that is how `pnpm run` behaves as well)

but when an error happens in pnpm CLI sub-process, no additional error should be printed. pnpm CLI prints all the necessary error info.

also, when executing binaries directly (like `pnpm eslint`), no additional comments should printed as well.

Reply:
> also, when executing binaries directly (like `pnpm eslint`), no additional comments should printed as well.

Did you mean `pnpx eslint`?

Reply:
no, we are going to deprecate `pnpx eslint`. `pnpm eslint` should work as `pnpx eslint` (without installing eslint).

Reply:
So, `eslint` is a special script, just like `test`, `start`, etc? Anyway, I don't see special scripts being handled by `pn`, so I will just ignore them for now.

Reply:
Here is how it should work:

`pnpm run eslint` runs the eslint script

`pnpm eslint` run the eslint script if available, if not available it should run eslint from `node_modules/.bin`